Berksfile

```diff
@@ -1,16 +1,13 @@
-source "https://api.berkshelf.com"
+source "https://supermarket.chef.io"
 
 metadata
 
-# until https://github.com/elastic/cookbook-elasticsearch/pull/230
-cookbook 'elasticsearch', '~> 0.3', git:'git@github.com:racker/cookbook-elasticsearch.git'
+cookbook 'elasticsearch', git: 'https://github.com/rackspace-cookbooks/rackspace_elasticsearch.git'
+cookbook 'logstash', git: 'https://github.com/rackspace-cookbooks/rackspace_logstash.git'
```

Comment:
whoops! I think I was wrong for pointing you at this ancient logstash cookbook repo. Looks like I had already switched back to lusis' and was pinned at a mythical 0.10 version that I can't find a tag for. Hash 3d7632612da325459242ea17ba02a2368a4e63a1.